pkg/manifest/anaconda_installer_iso_tree.go

```diff
@@ -390,6 +390,8 @@ keyboard us
 timezone UTC
 
 clearpart --all
+
+part biosboot --size=1 --fstype=biosboot
```

Comment:
I believe we should be able to use `reqpart` too now.

Reply:
Tracking here: #362.